src/modules/RootPart.rb

```diff
@@ -1127,11 +1127,8 @@ def MountFSTab(fstab, message)
         "ext3",
         "ext4",
         "btrfs",
-        "minix",
-        "reiserfs",
```

Comment:
just are you sure that we without this can report error that reiserfs is not supported?

Reply:
I see, let me check that...

Reply:
I turned out that I still have a SLES10 VM with ReiserFS, so I could test that.

And yes it works:

![sles10_upgrade_reiserfs](https://user-images.githubusercontent.com/907998/37462787-671227b8-2853-11e8-83a9-4ee0af7cdd6a.png)